src/pages/edit-item/edit-item.ts

```diff
@@ -129,6 +152,11 @@ export class EditItemPage {
     modal.present();
   }
 
+  /**
+   * Calls the api to create a new brand model or category, pushes it to the
+   * local brands, models or categories calls assignElement() to assign it to
```

Comment:
This comment seems too focused on the "how" over the "what" and "why." I think this one could be made more concise and the detail of calling `assignElement()` could be removed.